src/api-legacy/v1/borehole/export/spatialite.py

```diff
@@ -426,8 +423,6 @@ async def execute(self, workgroup=None, ids=None):
 
                         borehole['location_x'],
                         borehole['location_y'],
-                        borehole['location_x_lv03'],
-                        borehole['location_y_lv03']
```

Comment:
Q: Werden LV03 Koordinaten in Zukunft immer on the fly berechnet? Weshalb fliegen die columns hier raus?

Reply:
Beim neu Erstellen der LV03 Koordinaten haben wir im alten Backend überall wo es sinnvoll erschien die Werte ergänzt, der Export wurde aber nie entsprechend umgestellt (was auch nicht sinnvoll wäre, da er eh komplett neu erarbeitet wird). Die LV03 Einträge können also ohnehin nicht exportiert werden und führten zu einem Fehler, der bisher nicht aufgefallen ist.